src/Components/Wishlist/Wishlist.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import { WishlistCardItem } from "./WishlistCardItem";
+import { useStateProvider } from "../../Context/StateContext/state-provider";
+
+export const Wishlist = () => {
+  const { userState } = useStateProvider();
+  return (
+    <div>
+      <h3 className="text-center">My Wishlist </h3>
+
+      <div className="spacer-3rem"></div>
```

Comment:
Why add an extra `div` for spacing? 

You can give a margin to heading and second child